lib/cocoapods/installer/xcode/pods_project_generator/pod_target_installer.rb

```diff
@@ -644,7 +644,7 @@ def apply_xcconfig_file_ref_to_resource_bundle_targets(resource_bundle_targets,
           def create_module_map(native_target)
             return super(native_target) unless custom_module_map
 
-            path = target.module_map_path
+            path = Target.instance_method(:module_map_path).bind(target).call
```

Comment:
can you explain why this change?

Reply:
sure, will add this in a comment or factor this to make it less hideous:

We want always to store the actual file in the `Target Support Files` directory in the sandbox, but the `PodTarget#module_map_path` method refers to the path used to _import_ the module map in build settings, which needs to be next to the headers in question